kie-memory-compiler/src/main/java/org/kie/memorycompiler/jdknative/NativeJavaCompiler.java

```diff
@@ -86,6 +87,9 @@ public CompilationResult compile( String[] pResourcePaths,
                                       JavaCompilerSettings pSettings) {
         DiagnosticCollector<JavaFileObject> diagnostics = new DiagnosticCollector<JavaFileObject>();
         javax.tools.JavaCompiler compiler = ToolProvider.getSystemJavaCompiler();
+        if (compiler == null) {
+            throw new KieMemoryCompilerException("Cannot find the System's Java compiler. Please use JDK instead of JRE");
```

Comment:
What about mention `drools-ecj` in the exception as alternative?
```suggestion
            throw new KieMemoryCompilerException("Cannot find the System's Java compiler. Please use JDK instead of JRE or add drools-ecj dependency to use in memory Eclipse compiler");
```

Reply:
@danielezonca Thank you for the suggestion